### PR TITLE
Added persistent volume claim for etcd and config map updates

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: templates/run_template.sh [hunk 1]
```diff
@@ -6,9 +6,12 @@ OPENSHIFT_PASS="admin"
 DOCKERHUB_USER="CHANGEME"
 DOCKERHUB_PASS="CHANGEME"
 DOCKERHUB_ORG="ansibleplaybookbundle"
+REGISTRY_TYPE="dockerhub"
+REGISTRY_URL="docker.io"
+DEV_BROKER="true"
```
Comment (rthallisey) [inline templates/run_template.sh]: I think this depends on https://github.com/openshift/ansible-service-broker/pull/166

Comment (dymurray) [inline templates/run_template.sh]: Ah, thank you I will fix my config syntax too.

**Comment by jwmatthews** [comment]
@dymurray I tested deploying this from catasb, basic deployment checked out.

**Comment by rthallisey** [comment]
Ok just merged #166. @jwmatthews do you want to test again?

**Comment by jwmatthews** [comment]
I've tested what I can, deployment of broker/etcd is coming up.  Having issues with provision of APB through broker, but I think that is to be expected at moment.

I'm good with merging this and continue to debug.

**Pull request merged**